starlite/config.py

```diff
@@ -141,6 +142,20 @@ def dict(self, *args, **kwargs) -> Dict[str, Any]:  # type: ignore[no-untyped-de
         return super().dict(*args, **kwargs)
 
 
+class CSRFConfig(BaseModel):
```

Comment:
note, you can rebase your PR on my own PR which is about to be merged. In that PR there is a type `starlite.datastructures.Cookie`, which you could use as well. In either case - take a look there-->  https://github.com/starlite-api/starlite/pull/339

Reply:
@Goldziher do you want to do it in this PR or in a separate one?